shared/src/types/transaction/mod.rs

```diff
@@ -5,6 +5,8 @@
 pub mod decrypted;
 /// tools for encrypted data
 pub mod encrypted;
+/// txs to manage governance
```

Comment:
(nit) I see we've done this before, but we should really be putting the module level docs at the top of the governance.rs module, not here (module level docs start with `//!` - https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html)